pkgs/applications/networking/instant-messengers/schildichat/schildichat-desktop.nix

```diff
@@ -0,0 +1,128 @@
+{ lib
+, element-desktop # for seshat and keytar
+, schildichat-web
+, stdenv
+, fetchgit
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, fetchYarnDeps
+, yarn, nodejs, fixup_yarn_lock
+, electron
+, Security
+, AppKit
+, CoreServices
+
+, useWayland ? false
+}:
+
+let
+  pinData = lib.importJSON ./pin.json;
+  executableName = "schildichat-desktop";
+  electron_exec = if stdenv.isDarwin then "${electron}/Applications/Electron.app/Contents/MacOS/Electron" else "${electron}/bin/electron";
+in
+stdenv.mkDerivation rec {
+  pname = "schildichat-desktop";
+  inherit (pinData) version;
+
+  src = fetchgit {
+    url = "https://github.com/SchildiChat/schildichat-desktop/";
+    rev = "v${version}";
+    sha256 = pinData.srcHash;
+    fetchSubmodules = true;
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/element-desktop/yarn.lock";
+    sha256 = pinData.desktopYarnHash;
+  };
+
+  nativeBuildInputs = [ yarn fixup_yarn_lock nodejs makeWrapper copyDesktopItems ];
+  inherit (element-desktop) seshat keytar;
+
+  configurePhase = ''
+    runHook preConfigure
+
+    export HOME=$(mktemp -d)
+    pushd element-desktop
+    yarn config --offline set yarn-offline-mirror $offlineCache
+    fixup_yarn_lock yarn.lock
+    yarn install --offline --frozen-lockfile --ignore-platform --ignore-scripts --no-progress --non-interactive
+    rm -rf node_modules/matrix-seshat node_modules/keytar
+    ln -s $keytar node_modules/keytar
+    ln -s $seshat node_modules/matrix-seshat
+    patchShebangs node_modules/
+    popd
+
+    runHook postConfigure
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    pushd element-desktop
+    npx tsc
+    yarn run i18n
```

Comment:
@yu-re-ka

Reply:
Are you sure? Would this be translated without the overlays?

![image](https://user-images.githubusercontent.com/86169957/139586546-0fe19279-3665-4240-a8fe-78671d6f0af8.png)

Reply:
Or this:
![image](https://user-images.githubusercontent.com/86169957/139586735-5672e452-f32a-444f-8642-0797f654ad33.png)

Reply:
Or this:

![image](https://user-images.githubusercontent.com/86169957/139586942-3ea622ea-b53a-41b8-bae6-c9cb058ac9a2.png)

Reply:
Or is it just about the electron app?

Reply:
Yes, only strings about the electron app will be impacted if it's about element-desktop. It's probably no issue right now, but it might be in the future. Also I didn't take a close look on everything, element-web and matrix-react-sdk shouldn't have this under any circumstances.

Reply:
If you want to regenerate the strings (for whatever reason, this should actually only be needed if strings or code changed) you can run `regenerate_i18n.sh` in schildichat-desktop.